client/lib/promote-post/string.ts

```diff
@@ -148,6 +150,11 @@ const BlazePressStrings = () => {
 	translate( 'Start typing country, state or city to see available options' );
 	translate( 'No results found' );
 	translate( 'Search for locations' );
+	translate( 'Good for: ' );
```

Comment:
I'm not sure if this makes a difference but I'm not sure if we need space and punctuation here? `: `

Please ignore though if this doesn't matter.